.eslintrc.json

```diff
@@ -0,0 +1 @@
+{"root": true}
```

Comment:
we don't need this, having `root:true` in `.eslintrc.js` should be enough

Reply:
Removed it, thanks!